lib/core/config/settings.yml

```diff
@@ -94,3 +94,7 @@ smtp:
   ssl: false
   tls: false
   from: 'no-reply@example.com'
+
+bullet:
```

Comment:
As per our discussion, this is not needed and should be replaced with the settings in the config/deployment as i showed you